app/views/content/non-uk-teachers/non-uk-qualifications.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Check your eligibility to train to teach in England"
+description: |-
+  Find out what qualifications you need to train to teach in England. 
+related_content:
+    Learn more about fees and funding for non-UK trainees : "fees-and-funding-for-non-uk-trainees"
+    Train to teach in England as a non-UK citizen : "train-to-teach-in-england-as-an-international-student"
+    Learn more about international qualified teacher status : "international-qualified-teacher-status"
+promo_content:
+    - content/non-uk-teachers/promos/adviser-promo-training-in-england
+navigation: 20.20
+navigation_title: Qualifications you'll need to train to teach in England
+navigation_description: Before you apply, get help checking you have the correct qualifications for English teacher training.
+date: "2021-05-27"
+image: "static/images/content/hero-images/0003.jpg"
+backlink: "../../"
+youtube_video:
+  why-train-to-teach:
+    id: u0W7UTnGcc0
+    title: A video about why you should Train to Teach in England
+keywords:
+  - International
+  - Overseas
+  - Foreign
+  - Non-UK
+  - Nationality
+  - Residency
+  - EEA
+  - Canada
+  - New Zealand
+  - Switzerland
+  - USA
+  - Visa
+  - Skilled worker visa
+  - EU
+  - EU Settlement Scheme
+  - NARIC
+  - UK NARIC
+  - ENIC
+  - UK ENIC
+  - European Network of Information Centre
+  - National Recognition Information Centre for the United Kingdom
+  - PhD
+  - Immigration
+  - Ireland
+  - Irish
+  - Common Travel Area
+  - Student Visa
+  - Home Office
+  - Graduate Visa
+  - Youth Mobility Scheme
+  - Japan
+  - Monaco
+  - Hong Kong
+  - Republic of Korea
+  - Taiwan
+  - Family Visa
+  - UK Ancestry Visa
+  - Abroad
+  - Europe
+  - European
+  - Settlement Scheme
+  - Statement
+  - Comparability
+---
+
+Before you apply for teacher training in England, you must check that you have the [the right qualifications](#qualifications-you-will-need-to-train-to-teach) and they meet the standard set for English teacher training. 
+ 
+## How to validate your qualifications
+
+The simplest way to check your qualifications meet the required standard is to get a [statement of comparability](https://enic.org.uk/Qualifications/SOC/Default.aspx) from the UK European Network of Information Centres (UK ENIC). 
+
+This is a certificate costing £49.50 + VAT that proves your school and university qualifications are the same standard as UK GCSEs and a UK bachelor's degree. It takes 15 working days to complete your order from the date ENIC receives your qualification documents and payment.
+
+When you apply for teacher training, you'll be asked to submit details of your statement of comparability. Applications which include this are 28% more likely to be successful. 
```

Comment:
@laurahermionetennant92 would it be better to put the incentive higher up - applications are more likely to be successful if you get an ENIC - and then put the cost? Otherwise it may put people off and they may not carry on reading?

Reply:
yes good point!